frontend/src/scenes/onboarding/OnboardingProductConfiguration.tsx

```diff
@@ -14,53 +42,88 @@ export const OnboardingProductConfiguration = ({
     options: ProductConfigOption[]
 }): JSX.Element | null => {
     const { configOptions } = useValues(onboardingProductConfigurationLogic)
+    const { defaultEnabledPlugins } = useValues(pluginsLogic)
     const { setConfigOptions, saveConfiguration } = useActions(onboardingProductConfigurationLogic)
+    const { toggleEnabled } = useActions(pluginsLogic)
+
     useEffect(() => {
         setConfigOptions(options)
     }, [])
 
-    return configOptions ? (
+    const combinedList: ConfigOption[] = [
+        ...configOptions.map((option) => ({
+            title: option.title,
+            description: option.description,
+            type: option.type as ConfigType,
+            selectOptions: option.selectOptions,
+            value: option.value,
+            onChange: (newValue: boolean | string | number) => {
+                setConfigOptions(
+                    configOptions.map((o) => (o.teamProperty === option.teamProperty ? { ...o, value: newValue } : o))
```

Comment:
Note: there is an existing bug here for configs that have a select option like Session Replay. When changing the select value the `configOptions` value in the onChange callback the default one and overwrites any other changes made.